wrapper/tl.js

```diff
@@ -0,0 +1,12 @@
+const tljs = require('@y21/tljs');
+
+module.exports = function(html, callback) {
+    tljs.parse(html).then((dom) => {
+        callback();
+        dom.free();
```

Comment:
I'm not convinced having a free call after `callback` isn't cheating. Either move it to before or implement `WeakRef` in `@y21/tljs`. You should properly implement `WeakRef` no matter what to avoid a memory leak.